.github/workflows/build.yml

```diff
@@ -62,14 +62,18 @@ jobs:
           registry-url: "https://npm.pkg.github.com"
           scope: ${{ inputs.registry_scope }}
 
-      - name: Install Dependencies
+      - uses: actions/setup-node@v3.4.1
+        if: steps.s3-cache.outputs.processed == 'false'
+        with:
+          node-version: "16"
+          registry-url: "https://npm.pkg.github.com"
+          scope: "@pleo-io"
+
+      - name: Setup pnpm cache
         if: steps.s3-cache.outputs.processed == 'false'
```

Comment:
Note that the [exploration PR](https://github.com/pleo-io/product-web/pull/9390) didn't have the `if: steps.s3-cache.outputs.processed == 'false'` check, but I believe it should.